static-build/cloud-hypervisor/docker-build/build.sh

```diff
@@ -11,7 +11,13 @@ set -o pipefail
 script_dir=$(dirname $(readlink -f "$0"))
 docker_image="cloud-hypervisor-builder"
 
-sudo docker build -t "${docker_image}" "${script_dir}"
+DOCKER_CLI="docker"
+
+if ! command -v docker && command -v podman; then
```

Comment:
Nit: You might want to add `&>/dev/null` to both the `command` calls (here and below) to avoid the paths appearing in build logs?